### PR TITLE
Added first_contributors file for the new contributors

### PR DESCRIPTION
**Pull request by VipulRamtekkar**

### CODE REVIEW

File: new_contributor.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+## Fork this repository
+
+Fork this repository by clicking on the fork butoon on the top right of this page. This will create a copy of this repository in your account. Copy the url of the cloned repository from your account.
```
Comment (ritwik12) [inline new_contributor.md]: Correct typo "button"

**Comment by ritwik12** [comment]
@riya-17 Looks good to me after typo correction. Please test code for proper working.

**Comment by riya-17** [comment]
@VipulRamtekkar Thanks for the PR and sorry for late response.

**Comment by riya-17** [comment]
@ritwik12 Thanks for bringing my attention to this

**Pull request merged**